lms/templates/ux/reference/legacy/course-skeleton.html

```diff
@@ -0,0 +1,945 @@
+## mako
```

Comment:
Makes sense. However, I don't like the vague use of 'legacy'. I think the versioning scheme, or some other consistent label is better, because the pattern library is or is soon to be legacy as well.